packages/liquidator/src/liquidator.js

```diff
@@ -265,9 +265,28 @@ class Liquidator {
         inputPrice: scaledPrice.toString()
       });
 
-      // we couldnt liquidate, this typically would only happen if our balance is 0
+      // we couldnt liquidate, this typically would only happen if our balance is 0 or the withdrawal liveness
+      // has not passed the WDF's activation threshold.
       // This gets logged as an event, see constructor
       if (!liquidationArgs) {
+        // If WDF is active but liveness hasn't passed activation %, then send customized log:
+        if (
+          this.defenseActivationPercent &&
+          !this.liquidationStrategy.utils.passedDefenseActivationPercent({ position, currentBlockTime })
+        ) {
+          this.logger.info({
+            at: "Liquidator",
+            message: "Withdrawal liveness has not passed WDF activation threshold, skipping✋",
```

Comment:
shall we use a new emoji? this one is used in other places.

Reply:
key comment, will do